src/controllers/PaymentController.ts

```diff
@@ -12,11 +12,11 @@ export default class PaymentController {
         }
     }
 
-    public async deletePayment(paymentId: string): Promise<void> {
+    public async deletePayment(paymentId: Object|undefined): Promise<void> {
```

Comment:
if we allow paymentID of undefined, this will lead to an error since you're not handling undefined IDs